editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx

```diff
@@ -40,7 +56,10 @@ function Component(props: Props) {
               Required files
             </Typography>
             {props.fileTypes.map((fileType) => (
-              <p key={fileType.key}>{fileType.key}</p>
+              <InteractiveFileListItem
+                name={fileType.key}
```

Comment:
worth noting that React throws this error when trying to pass the full `fileType` as props to a child component: 
```
"key" is not a prop. Trying to access it will result in "undefined" being returned. If you need to access the same value within the Child component, you should pass it as a different value
```
we might want to consider renaming `key` to `name` in the `FileType` model too??

Reply:
Ah this is interesting and a good catch - I'd certainly be in favour of just changing to `name` if we end up having to do an awkward half mapping of key -> name to work around this 👍